### PR TITLE
Finished the excel data extraction. 

### PR DESCRIPTION
**Pull request by apennella**
I need to change 1 small functionality of the program, but this does work as long as the columns are set up in the excel sheet as : EmpId, EmpNam, Email, ClassID(String), Mod1(String).

### CODE REVIEW

File: Employee.java [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+	private String employeeEmail;
+	private double classID;
+	private List<Module> modScores = new ArrayList<>();	
+	
+	
+	
+	
+	public Employee() {
+		super();
+		// TODO Auto-generated constructor stub
```
Comment (kabollinger03) [inline Employee.java]: todo - fill out blank stub :)

File: Employee.java [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+	}
+	
+	@Override
+	public String toString() {
+		return "Employee [employeeID=" + employeeID + ", employeeName=" + employeeName + ", employeeEmail="
+				+ employeeEmail + ", classID=" + classID + ", modScores=" + modScores + "]";
+	}
+	
+	
+
```
Comment (kabollinger03) [inline Employee.java]: Employee has no other methods? OK I will probably find out why later.

File: Module.java [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+
+
+	@Override
+	public String toString() {
+		return "Module [moduleID=" + moduleID + ", moduelsScore=" + moduelsScore + "]";
+	}
+	
+	
+
```
Comment (kabollinger03) [inline Module.java]: Whitespace overload much?

File: Puller.java [hunk 4]
```diff
@@ -17,46 +18,107 @@
 import java.util.Iterator;
 public class Puller {
 	
-
-
+	/*
```
Comment (kabollinger03) [inline Puller.java:21]: Don't leave comments like this in the final code. Delete them. Also this is where you should make a PR to the sub branch. Make your changes, PR them to the branch. Leave comments there.

File: Puller.java [hunk 5]
```diff
@@ -17,46 +18,107 @@
...
+	 * 	I set the currentRow in the 2nd while loop to get the cell values
+	 * 	I added a few more lines, mostly just print statements and comments
+	 * 	You should run it to see what it outputs.
+	 * 
+	 * --Chris "Big Data" Mendoza
+	 */
+	
             private static final String FILE_NAME = "C:\\Users\\syntel\\Downloads\\MockData.xlsx";
 
             @SuppressWarnings("deprecation")
```
Comment (kabollinger03) [inline Puller.java:32]: Good enough for now, but should we really be using deprecated stuff?

File: Puller.java [hunk 6]
```diff
@@ -17,46 +18,107 @@
...
                 } catch (FileNotFoundException e) {
                     e.printStackTrace();
                 } catch (IOException e) {
                     e.printStackTrace();
+                } finally {
+                	
+                	 for(int i = 0; i < emps.size(); i++) {   
+                         System.out.println(emps.get(i) + " ");
+                     } 
+                	workbook.close();
```
Comment (kabollinger03) [inline Puller.java:118]: This should be inside the finally block. Or even better, use a try-with-resources and declare workbook under it

**Review by kabollinger03 — DISMISSED** [review]
Inconsistent whitespace. Overused whitespace. A lot of other stuff that is just me being pedantic.

**Comment by kabollinger03** [comment]
> I need to change 1 small functionality of the program, but this does work as long as the columns are set up in the excel sheet as : EmpId, EmpNam, Email, ClassID(String), Mod1(String).

That is fine because we are making them download a template to fill. In fact you can use that breakage as a check...

File: Employee.java (hunk 2 above)
Comment (apennella) [reply Employee.java]: Employee should have all the getters and setters in it. Plus the toStri g and constructors. For now that is all I need to do what I need the class to. Which is pull the data from the excel. Then transfer it to the appropriate Crud functions.

File: Puller.java (hunk 4 above)
Comment (apennella) [reply Puller.java:21]: I just forgot to take it out. I'll take it out when I next up date it.

File: Puller.java (hunk 5 above)
Comment (apennella) [reply Puller.java:32]: I wasnt sure what that meant. For now it works good enough for the prototype though, and we can fix that layer

File: Puller.java (hunk 6 above)
Comment (apennella) [reply Puller.java:118]: It was suppose to be. I'll have to fix that as well. Idk how it got outside

File: Module.java [hunk 7]
```diff
@@ -0,0 +1,36 @@
+package extraction.sProject;
+
+public class Module {
+	
+	private String moduleID;
+	private double moduelsScore;
```
Comment (ChrisSMendoza) [inline Module.java:6]: You put "moduelsScore" instead of "moduleScore" or something like this. Your getters and setters use this value too.

**Pull request closed without merging**